stable/rook-ceph/values.yaml

```diff
@@ -0,0 +1,294 @@
+# Default values for rook-ceph-operator
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  prefix: rook
+  repository: registry.suse.com/ses/7/rook/ceph
+  tag: 1.4.2
```

Comment:
there probably should be container that is available for this chart. currently we have 1.4.0 only.

Reply:
Right - I've got 1.4.2 ready to go though, and users can overwrite this with whatever version they want.

Reply:
I think that is exactly the point that a default here is latest recommended value :) Cool, hope it would be there together with helm chart.

Reply:
Rook 1.4.2 and ceph-csi 3.1.0 have been submitted, so they should be in sync now.